messaging.py

```diff
@@ -0,0 +1,79 @@
+from multiprocessing import Array, Value
+import cPickle as pickle
+import logging
+import sys
+
+
+class Messenger(object):
+  """
+    Manages messaging between local processes
+  """
+
+  class Queues(object):
+    """
+      Enum for messenger queues
+    """
+    uavStatus = 'uavStatus'
+    toBelief = 'toBelief'
+    fromRadio = 'fromRadio'
+
+  class SingleQueue(object):
+    """
+      Lockable queue with only one element
+    """
+
+    # The size of the array that stores data.
+    _SIZE = 1024
+
+    def __init__(self, name):
+      # Where we write the element.
+      self.__state = Array('c', self._SIZE)
```

Comment:
Well, it was never really a queue TBH. It wasn't necessary, but I figured that using an actual queue to implement a much simpler data structure was overkill.